scripts/quests/windurst/RNG_AF1_Sin_Hunting.lua

```diff
@@ -0,0 +1,176 @@
+-----------------------------------
+-- Sin Hunting (RNG AF1)
+-----------------------------------
+-- Log ID: 2, Quest ID: 72
+-- Perih Vashai: !pos 117 -3 92 241
+-- Perchond: !pos -182.844 4 -164.948 166
+-- Alexis: !pos 105 1 382 104
+-- qm2: !pos -10.946 -1.000 313.810 104
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.WINDURST, xi.quest.id.windurst.SIN_HUNTING)
+
+quest.reward =
+{
+    item     = xi.item.SNIPING_BOW,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+            player:getMainLvl() >= xi.settings.main.AF1_QUEST_LEVEL and
+            player:getMainJob() == xi.job.RNG
+        end,
+
+        [xi.zone.WINDURST_WOODS] =
+        {
+            ['Perih_Vashai'] = quest:progressEvent(523),
+
+            onEventFinish =
+            {
+                [523] = function(player, csid, option, npc)
+                    quest:begin(player)
+                    npcUtil.giveKeyItem(player, xi.ki.CHIEFTAINNESSS_TWINSTONE_EARRING)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.WINDURST_WOODS] =
+        {
+            ['Perih_Vashai'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') <= 3 then
+                        return quest:event(524)
+                    else
+                        return quest:progressEvent(527)
+                    end
+                end,
+            },
+
+            ['Kapeh_Myohrye'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') <= 4 then
```

Comment:
Looks like 4 is the maximum value used in this script.  Since this will always be true, the condition is not necessary here or below with Muhk Johldy

Reply:
So you are saying remove the if and just put return event ()?

Reply:
Since there is no other logic for these NPCs, in this example, 
```
            ['Kapeh_Myohrye'] =
            {
                onTrigger = function(player, npc)
                    if quest:getVar(player, 'Prog') <= 4 then
                        return quest:event(526)
                    end
                end,
            },
```
can be simplified to:
`['Kapeh_Myohrye'] = quest:event(526),`

Reply:
Ok - thats what I was trying to say above. Got it - will fix shortly.

Reply:
@claywar Thanks- complete.